stdpopsim/catalog/drosophila_melanogaster.py

```diff
@@ -45,9 +56,10 @@
     id="dromel",
     name="Drosophila melanogaster",
     genome=_genome,
-    # TODO reference for these
     generation_time=0.1,
-    population_size=10**4)  # TODO: This is wildly wrong, right?
+    generation_time_citations=[_LiAndStephan],
+    population_size=8.603e06,
```

Comment:
This is the current size of the African population, and the ancestral size was inferred to be 1/5 the current size. Do we want to use the current Ne from this paper, or the ancestral size (1.72e6)?

Reply:
i think we'd want to use the current size?

Reply:
My thought was that the ancestral size probably better reflects average heterozygosity (similar to using Ne=1e4 in humans even though it's much smaller than current sizes). Practically, the smaller size probably is easier to simulate with a recombination map (are those large rho values still bogging down dmel simulations)? But I'm fine with either choice, and things look good otherwise :)

Reply:
yeah rho still bogs things down. happy to go with the smaller estimate, as I think you are right-- it would be ideal if the defaults ran relatively fast

Reply:
okay made that change @apragsdale